### PR TITLE
feat: Update Toast Notification to handle long strings KZN-1005

### PR DESCRIPTION
**Pull request by NatalieLoyzaga**
## Why

- Wrap long single words within the title - most likely caused through translations

## What

- Update the text container to include a max-width and allow word breaks

### CODE REVIEW

File: packages/notification/docs/ToastNotification.stories.tsx [hunk 1]
```diff
@@ -224,6 +224,20 @@ const StickerSheetTemplate: Story = () => {
       ),
       autohide: !IS_CHROMATIC,
     })
+    addToastNotification({
+      type: "positive",
+      title: "VerylongTitleExampleVerylongtitleExample ",
```
Comment (gyfchong) [inline packages/notification/docs/ToastNotification.stories.tsx:229]: Needs to be longer!
image

**Review by gyfchong — DISMISSED** [review]

**Comment by NatalieLoyzaga** [comment]
@gyfchong I had to update this so that the styles only target the `Toast Notification` - Chromatic picked up changes on the `Inline Notification`

File: packages/notification/src/_styles.scss [hunk 2]
```diff
@@ -201,6 +201,13 @@ $notification-slide-right: transform 300ms ease-out;
     margin-left: 6px;
   }
 
+  %ca-notification---toast & {
+    margin-left: 6px;
+    max-width: calc(100% - #{$spacing-xl});
+    overflow-wrap: break-word;
+    hyphens: auto;
```
Comment (gyfchong) [inline packages/notification/src/_styles.scss:208]: Maybe we can apply these styles based on a `@container` query? I'm not entirely sold on having a `isToast` prop for this 😅

Comment (NatalieLoyzaga) [reply packages/notification/src/_styles.scss:208]: Hm Im getting a lint issue when trying to use it - unknown at rule container css(unknownAtRules)

Comment (gyfchong) [reply packages/notification/src/_styles.scss:208]: Probably worth digging a little into why, we'll need this for future work too. If it ends up being too deep, make a ticket and we can move on

Comment (NatalieLoyzaga) [reply packages/notification/src/_styles.scss:209]: I added a container query - thoughts?

Comment (gyfchong) [reply packages/notification/src/_styles.scss:209]: That's like magic lol

**Review by gyfchong — APPROVED** [review]

**Pull request merged**